k8s/openebs-monitoring/prometheus.yaml

```diff
@@ -0,0 +1,169 @@
+# ConfigMap is used to run prometheus with given configuration. It helps
+# if we want to change the configuration from time to time because our 
+# requirement changes so configurations also need to change accordingly.
```

Comment:
Does change in ConfigMap involve any restart ? Please mention if restart or no restart.